update.py

```diff
@@ -214,6 +240,8 @@ def download_zzzs_xlsx_files():
 
 
 if __name__ == "__main__":
+    get_overrides()
+    append_overrides()
```

Comment:
Move this lower, as it will be overwritten in convert_to_csv().

Reply:
It was there just for development so that it happened on start of the script :)